SEED-platform/seed
# 3188 bug/derived cols letter spacing

#### Any background context you want to provide?
When viewing the inventory list, derived columns with all caps titles appeared with spaces inserted between each consecutive capital letter: `TEST column` is displayed as `T E S T column`. 

#### What's this PR do?
This is an angular ui-grid "feature". If a column does not have a `.displayName` then one will be generated and spaces will be inserted between consecutive capital letters. When the inventory lists are loaded in the controller, the derived columns will be assigned a `.displayName` based on `.name`

#### How should this be manually tested?
- Generate a derived column with at least one all capital word in the title.
- Go to Column List Profile and enable `√` the new derived column
- Go to inventory or Inventory (beta) and ensure the title of the derived column does not have spaces inserted between each capital letter

#### What are the relevant tickets?
#3188

#### Screenshots (if appropriate)
NEW
![Screen Shot 2022-04-01 at 1 17 30 PM](https://user-images.githubusercontent.com/58446472/161327782-2cdb9098-caeb-4b7f-a2d2-21d231756c42.png)

OLD
![Screen Shot 2022-04-01 at 1 16 54 PM](https://user-images.githubusercontent.com/58446472/161327787-a4891a96-8fc3-4917-9e57-6571e65c853a.png)
